pkg/controller/storagecluster/controller_test.go

```diff
@@ -5043,13 +5043,6 @@ func TestUpdateStorageClusterNodeSpec(t *testing.T) {
 	err = k8sClient.Update(context.TODO(), cluster)
 	require.NoError(t, err)
 
-	// Change existing pod's hash to latest revision, to simulate new pod with latest spec
```

Comment:
when pod is updated without restart, the hash update will be handled in the code, so no simulation is required.